tests/integration/framework/test_dashboards.py

```diff
@@ -15,7 +18,7 @@
 # logging.getLogger("databricks").setLevel("DEBUG")
 
 
-def test_creating_widgets(ws: WorkspaceClient):
+def test_creating_widgets(ws: WorkspaceClient, make_warehouse, make_schema):
```

Comment:
You can use the preprovisioned warehouse through environment variable, no need to create new warehouse to test a dashboard.